src/main/java/org/edumips64/utils/cli/FileCommand.java

```diff
@@ -0,0 +1,61 @@
+package org.edumips64.utils.cli;
+
+import org.edumips64.core.CPU;
+import org.edumips64.core.parser.ParserMultiException;
+import org.edumips64.utils.CurrentLocale;
+import org.edumips64.utils.io.ReadException;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+import picocli.CommandLine.Parameters;
+import picocli.CommandLine.Option;
+
+import java.io.File;
+
+
+@Command(name = "file", resourceBundle = "CliMessages")
+public class FileCommand implements Runnable {
+
+    @ParentCommand
+    private Cli cli;
+
+    @Option(names={"-h","--help"}, usageHelp=true, descriptionKey = ".help.usage.description")
+    private boolean help;
+
+    @Parameters(arity = "1", paramLabel = "file_to_run", descriptionKey = "file.param")
+    private File file;
+
+    @Override
+    public void run() {
+        String absoluteFilename = file.getAbsolutePath();
+        try {
+            if (canLoadFile()) {
+                cli.getParser().parse(absoluteFilename);
+                cli.getDinero().setDataOffset(cli.getMemory().getInstructionsNumber() * 4);
+                cli.getCPU().setStatus(CPU.CPUStatus.RUNNING);
+                printLoadedFileMsg(absoluteFilename);
+            } else {
+                printCannotLoad();
+            }
+        } catch (ParserMultiException | ReadException e) {
```

Comment:
The `ParserMultiException` contains all the parsing errors encountered by the simulator, and they are necessary for the user to understand how to fix their code.

Would it be possible to output the parsing errors?

Reply:
Definitely, looking inside ParserMultiException, it appears there is a good toString(). Is that preferred for outputting the parser errors?

Reply:
I think that should be ok, yes! We might need to tweak it a bit (not sure if that correctly distinguishes warnings vs. errors), I'd need to test it to be sure :) But as a first step, that's definitely good enough!